e-sensing/sits
# Add fixtures tests

- Add fixtures to tests bdc cubes
- Add more tests
- Translate assertthat to .check in sits
- Update DESCRIPTION
